lib/http_signature.rb

```
@@ -77,20 +79,21 @@ def self.create(
       end
 
     canonical_components = build_components(
-      uri: uri,
-      method: method,
+      uri:,
```

RuboCop (1.82.1)

[fatal] unexpected token tCOMMA
(Using Ruby 3.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)